wiki/History_of_osu!/en.md

```diff
@@ -7,18 +7,23 @@ needs_cleanup: true
 
 Records of osu!'s history, all in the osu! wiki.
 
-- [2007](2007) - Initial game build
-- [2008](2008) - Taiko and Catch the Beat, more mods, and major improvements
-- [2009](2009) - osu! iPhone
-- 2010
-- [2011](2011) - osu! wiki
-- [2012](2012) - Five years of osu!, osu!mania, widescreen support, and visual settings
-- [2013](2013) - UI overhaul, HD skins, improved widescreen support, osu!mania editor, in-game collections, moddingV2 was introduced, Disqus, osu!api, accuracy overlays, Comic Fiesta 2013, downloads unrestricted, and osu!cuttingedge (osu!supporters only)
-- [2014](2014) - BPM scrolling, in-game intro/outro sequences, osu!cuttingedge (for everyone), and snow!
-- [2015](2015) - jizz (redesigned osu!web), osu!weekly, and osu!coins
-- [2016](2016) - scorev2, announcement of a new open-source "lazer" version of osu! (previously known as "osu!lazer"), and an osu! wiki overhaul
-- [2017](2017) - release of the first lazer build of osu!, and Discord integration on the stable client
-- [2018](2018)–present - ?
+| Year | Description |
+| :-: | :-- |
+| [2007](2007) | Initial game build |
+| [2008](2008) | Taiko and Catch the Beat, more mods, and major improvements |
```

Comment:
```suggestion
| [2008](2008) | osu!taiko and osu!catch (then Taiko and Catch the Beat), more mods, and major improvements |
```

Reply:
I think it is better to move the name change explanation to the article (probably as a notice) and just call them their current name here.